user/admin.py

```diff
@@ -1,5 +1,40 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext as _
+
 from .models import User
 
-admin.site.register(User, UserAdmin)
+
+@admin.register(User)
+class UserAdmin(DjangoUserAdmin):
+    """Define admin model for custom User model with no email field."""
```

Comment:
The docstring mentions 'no email field', but the email field is actually included in both `fieldsets` and `add_fieldsets`. Consider updating the docstring to accurately reflect the fields used in this admin model.